pkg/util/export/example/main.go

```diff
@@ -0,0 +1,236 @@
+	execPlanCol  = export.Column{Name: "exec_plan", Type: "JSON", Default: jsonColumnDEFAULT, Comment: "statement execution plan"}
+	rowsReadCol  = export.Column{Name: "rows_read", Type: bigintUnsignedType, Default: "0", Comment: "rows read total"}
+	bytesScanCol = export.Column{Name: "bytes_scan", Type: bigintUnsignedType, Default: "0", Comment: "bytes scan total"}
+
+	dummyStatementTable = &export.Table{
+		Account:  export.AccountAll,
+		Database: statsDatabase,
+		Table:    statementInfoTbl,
+		Columns: []export.Column{
+			stmtIDCol,
+			txnIDCol,
+			sesIDCol,
+			accountCol,
+			userCol,
+			hostCol,
+			dbCol,
+			stmtCol,
+			stmtTagCol,
+			stmtFgCol,
+			nodeUUIDCol,
+			nodeTypeCol,
+			reqAtCol,
+			respAtCol,
+			durationCol,
+			statusCol,
+			errCodeCol,
+			errorCol,
+			execPlanCol,
+			rowsReadCol,
+			bytesScanCol,
+		},
+		PrimaryKeyColumn: []export.Column{stmtIDCol},
+		Engine:           export.ExternalTableEngine,
+		Comment:          "record each statement and stats info",
+		PathBuilder:      export.NewAccountDatePathBuilder(),
+		AccountColumn:    &accountCol,
+		// SupportUserAccess
+		SupportUserAccess: true,
+	}
+
+	rawItemCol      = export.Column{Name: "raw_item", Type: stringType, Comment: "raw log item"}
+	timestampCol    = export.Column{Name: "timestamp", Type: datetime6Type, Comment: "timestamp of action"}
+	loggerNameCol   = export.Column{Name: "logger_name", Type: stringType, Comment: "logger name"}
+	levelCol        = export.Column{Name: "level", Type: stringType, Comment: "log level, enum: debug, info, warn, error, panic, fatal"}
+	callerCol       = export.Column{Name: "caller", Type: stringType, Comment: "where it log, like: package/file.go:123"}
+	messageCol      = export.Column{Name: "message", Type: "TEXT", Comment: "log message"}
+	extraCol        = export.Column{Name: "extra", Type: "JSON", Default: jsonColumnDEFAULT, Comment: "log dynamic fields"}
+	errCodeCol      = export.Column{Name: "err_code", Type: stringType, Default: "0"}
+	stackCol        = export.Column{Name: "stack", Type: "varchar(4096)"}
+	spanIDCol       = export.Column{Name: "span_id", Type: spanIDType, Default: "0", Comment: "span uniq id"}
+	parentSpanIDCol = export.Column{Name: "parent_span_id", Type: spanIDType, Default: "0", Comment: "parent span uniq id"}
+	spanNameCol     = export.Column{Name: "span_name", Type: stringType, Default: "", Comment: "span name, for example: step name of execution plan, function name in code, ..."}
+	startTimeCol    = export.Column{Name: "start_time", Type: datetime6Type, Default: ""}
+	endTimeCol      = export.Column{Name: "end_time", Type: datetime6Type, Default: ""}
+	resourceCol     = export.Column{Name: "resource", Type: "JSON", Default: jsonColumnDEFAULT, Comment: "static resource information"}
+
+	dummyRawlogTable = &export.Table{
+		Account:  export.AccountAll,
+		Database: statsDatabase,
+		Table:    rawLogTbl,
+		Columns: []export.Column{
+			rawItemCol,
```

Comment:
this column list is different from listed above.   intentional?    should be documented.

Reply:
yes. i add more note in code. doc can see [link](https://github.com/matrixorigin/docs/blob/main/design/observability/log_schema.md)